meson.build

```diff
@@ -0,0 +1,53 @@
+project('facil', 'c',
+    version : '0.7.5',
+    license: 'MIT',
+    default_options: ['c_std=c11']
+)
+
+cc = meson.get_compiler('c')
+m_dep = cc.find_library('m', required : true)
```

Comment:
A portable way to add `m` is to use `required : false` because it's not always present as a standalone library: https://mesonbuild.com/howtox.html#add-math-library-lm-portably

Reply:
Thank you for this nice info